lib/deep-speech-interface.js

```diff
@@ -228,10 +238,7 @@ class DeepSpeechInterface {
     }, 3 * 1000);
 
     const dataHandler = (data) => {
-      this._model.feedAudioContent(
-        dsStream,
-        data.slice(0, data.length / 2)
-      );
+      dsStream.feedAudioContent(data.slice(0, data.length / 2));
```

Comment:
I'm not sure this is right

Reply:
https://github.com/mozilla/DeepSpeech/blob/r0.8/native_client/javascript/client.ts#L132

Reply:
The only difference here is the `slice()` bit, but it doesn't seem to make a difference either way.

Reply:
A slice where we cut in half is not a simple difference and explain the issue

Reply:
Ok, I stopped slicing, but still no STT results.